css/layout.css

```diff
@@ -0,0 +1,56 @@
+body {}
+
+main {
+    min-width: 480px;
+    margin: 0 auto;
+    max-width: 767px;
+}
+
+.col-2 {
+    width: 50%;
+    float: left;
+}
+
+footer {
+    clear: both;
+}
+
+@media (min-width:768px) {
+
+    main {
+        max-width: 960px;
+        min-width: 960px;
```

Comment:
It is often nicer to allow a range between the min-width and max-width to allow the page to adapt the size the user prefers. If your intent is to make the page exactly this size, then it is better to use the width rule. When the max-width and min-width are the same value it creates an opportunity for odd and unexpected consequences.